pkg/virt-operator/webhooks/kubevirt-update-admitter.go

```diff
@@ -162,3 +176,116 @@ func validateCertificates(certConfig *v1.KubeVirtSelfSignConfiguration) []metav1
 
 	return statuses
 }
+
+//validateWorkloadPlacement
+func validateWorkloadPlacement(placementConfig *v1.NodePlacement, client kubecli.KubevirtClient) []metav1.StatusCause {
+	statuses := []metav1.StatusCause{}
+
+	const (
+		dsName    = "placement-validation-webhook"
+		namespace = "default"
```

Comment:
@zcahana You are correct! thanks. I've decided to derive the namespace from the KV resource object that is passed to the webhook callback context. Since in KV we don't have the option to configure a scenario where the KV resource is in different NS then the components, we can safely assume that the NS provided in the KV object would exist and the webhook would have the permissions to do the dry-run in it.

Reply:
That makes sense. Great solution!